bin/XMLToSARIF.sh

```diff
@@ -48,28 +50,30 @@ getJavaPath() {
 BASE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 SARIF_XSL_PATH="$BASE_DIR/xsl/sarif.xsl"
 
-java_or_parasoft_tool_root_path=""
+parasoft_tool_or_java_root_path=""
 xml_report_path=""
 sarif_report_path=""
 
 # 2. Process options into variables.
 while [[ $# -gt 0 ]] ; do
-  case "$1" in
-    --inputXmlReport)
-        xml_report_path="$2"
-        check_param "$1" "$xml_report_path"
+  param="$1"
+  value="$2"
```

Comment:
In Git Bash, it seems that using `./xxx` to reference current directory is not supported for file path options. 
We may want to support this since it's a common usage.